src/SymmetricIntervalHull.jl

```diff
@@ -106,14 +106,14 @@ This function computes the symmetric interval hull explicitly.
 """
 function radius_hyperrectangle(sih::SymmetricIntervalHull{N}
                               )::Vector{N} where {N<:Real}
-    dim = dim(sih)
+    n = dim(sih)
     one_N = one(N)
-    for i in 1:dim
-        if !sih.cache[i][2]
-            compute_radius(sih, i, dim, one_N)
+    for i in 1:n
+        if sih.cache[i] == -1
+            compute_radius(sih, i, n, one_N)
```

Comment:
better if we call it `compute_radius!`, since it modifies `sih` in-place

Reply:
i would suggest that `compute_radius!` also returns the computed result, if not only for testing/debugging purposes

```julia
    sih.cache[i] = max(right_bound[i], abs(left_bound[i]))
end
```

Reply:
Good suggestions. I renamed `compute_radius!` to `get_radius!` and added the cache look-up inside this function, which made the code simpler.